documentation/ci-fix.md

```diff
@@ -49,6 +50,20 @@ Then please commit and push changes made by prettier.
 
 Reference: [prettier](https://www.npmjs.com/package/prettier).
 
+### Troubleshotting `npx prettier`
+
+If you see failure like the following upon running `npx prettier --check` or `npx prettier --write`:
+```
+[error] Cannot find module 'prettier/parser-babylon'
+[error] Require stack:
+[error] - azure-rest-api-specs-pr\scripts\prettier-swagger-plugin.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\index.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\cli.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\bin-prettier.js
+```
+
+then ensure you ran `npx tsc` from `main` branch and did not run `npx tsc` from any other branch, like `RPSaaSDev`.
```

Comment:
If folks are working in another branch what do you expect them to do? I think we need to figure out how to make this work correctly in any of our primary branches.

Reply:
I agree, but maybe we could update these instructions for now as a stop-gap solution?

Reply:
I added clarification.

Reply:
@weshaggard per our chat, you will take point on figuring out why the error is happening on the dev branch and override it to fix the npx tsc on it (see file diff for the error).